core/tee/tee_sql_fs.c

```diff
@@ -528,14 +528,13 @@ static TEE_Result sql_fs_read(struct tee_file_handle *fh, void *buf,
 static TEE_Result sql_fs_write(struct tee_file_handle *fh, const void *buf,
 			       size_t len)
 {
-	TEE_Result res;
+	TEE_Result res = TEE_SUCCESS;
```

Comment:
You're right, it's not needed. @jenswi-linaro does you R-b still hold if I remove the initialization?